moveit_planners/ompl/ompl_interface/demos/generate_state_database.cpp

```diff
@@ -118,20 +119,22 @@ void computeDB(const planning_scene::PlanningScenePtr& scene, struct GenerateSta
   scene->getCurrentStateNonConst().update();
 
   ompl_interface::OMPLInterface ompl_interface(scene->getRobotModel());
+  const planning_interface::MotionPlanRequest req;  // TODO(brycew): fill?
```

Comment:
seems like this works and does not need to be filled in. are you sure you need this TODO?

Reply:
Not quite, leaving those empty caused a segfault in the script because the context wouldn't have been built properly. Just fixed now.